ts/parts/SVGRenderer.ts

```diff
@@ -1620,6 +1620,9 @@ class SVGRenderer {
             ),
             curState = 0,
             styledMode = this.styledMode,
+            // Make a copy of normalState (#13798)
+            // (reference to options.rangeSelector.buttonTheme)
+            normalState = normalState ? H.merge(normalState, {}) : normalState,
```

Comment:
The merge can be simplified to `merge(normalState)` I think. Also, `merge` is already available in the local scope, so we don't need to access it through `H`.